docs/src/pages/components/skeleton/Children.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
```

Comment:
I am not sure what the guidelines are for PRs with demos. I don't have the TypeScript versions in this PR, but it's what I write normally, so no worries if you want me to push them as well.

Reply:
At this point, we ask all new demos to be written in TypeScript. I have added it :)